src/txn/txn_rollback_to_stable.c

```diff
@@ -7,7 +7,8 @@
  */
 
 #include "wt_internal.h"
-
+#define WT_RECOVERY_VALID_TXNID(session, txnid) \
```

Comment:
I didn't clearly understand the name of this macro. Update it according to the conditions.

Reply:
Fixed

Reply:
Updated the macro name